ArduCopter/mode.h

```diff
@@ -1140,7 +1144,8 @@ class ModeGuided : public Mode {
         DoNotStabilizePositionXY = (1U << 4),
         DoNotStabilizeVelocityXY = (1U << 5),
         WPNavUsedForPosControl = (1U << 6),
-        AllowWeatherVaning = (1U << 7)
+        AllowWeatherVaning = (1U << 7),
+        ArmingCheckThrottle = (1U << 8),
```

Comment:
Needs param description.

Reply:
Fixed, thanks.

We really should add a sanity check for that.